letter-shop/index.md

```diff
@@ -0,0 +1,634 @@
+
+### Client
+
+The client will perform an ILP payment and output the returned fulfillment. This can be redeemed to get the letter that was requested in the original request.
+
+Once we have completed the tutorial the client will perform the following steps:
+
+1. Connect to an account from which to send payments
+1. Initiate a payment using the address, amount, and condition provided in the response to the original letter request
+1. Complete the payment and return the proof-of-payment token to be used to complete the original request
+
+## Step 1: Get the code
+
+The code for this tutorial is available in two forms:
+
+1. If you follow the tutorial you will start with some very basic code and will flesh it out as we go. 
+2. If you'd prefer to start with the finished code and simply walk through the steps then you can do that too.
+
+First let's get all the code, either using Git or by simply downloading and unzipping it.
+
+#### Git
+
+```shell
+git clone https://github.com/interledger/tutorials
+```
+
+#### Download
+
+[ZIP Archive](https://github.com/interledger/tutorials/archive/master.zip)
+
+Assuming you have either cloned the repository, or unzipped your download, open a terminal window at the root of the project and change into the `letter-shop` directory.
+
+You should see the following files:
+
+```shell
+README.md
+package.json
+pay.js
+plugins.js
+proxy.js
+shop.js
+```
+
+Most of these files are just scaffolding, waiting for you to complete as we go. If you want the complete files so you can just follow along, switch into the `complete` sub-directory.
+
+## Step 2: Run the server
+
+The first thing we need to do is install our dependencies and start our shop's server.
+
+We install using npm (this may take a few minutes while it downloads all the dependencies):
+
+```shell
+npm install
+```
+
+> If you get an error here make sure you have `node` installed.
+
+Then we try and start our shop server:
+
+```shell
+node shop.js
+```
+
+But, we've hit a snag... Read on.
+
+### Interledger plugins
+
+Our shop, and many other components we'll build in this tutorial, use ledger plugins. A plugin is a piece of code that talks to a specific account on a specific ledger.
+
+Since Interledger connects potentially very different ledgers together, we need an abstraction layer that hides the specifics of the ledger, but exposes the interface needed to send and receive money over that ledger. That is what Interledger plugins are for. 
+
+Ledger plugins expose a common interface (the Ledger Plugin Interface) so that irrespective of which ledger your application is connected to, the way it sends and receives payments is identical.
+
+> **NOTE:** As we go through this tutorial we'll be using different functions of the Ledger Plugin Interface. You can find the reference documentation in [IL-RFC 0004](https://interledger.org/rfcs/0004-ledger-plugin-interface/).
```

Comment:
You removed the `/draft-8.html` from the hyperlink, was that intentional? I think it's quite important we quote specific versions of all the RFCs (then we can control how and when we update this tutorial to future RFC versions)